src/gui/transferlistmodel.cpp

```diff
@@ -709,52 +709,46 @@ QColor getDefaultColorByState(const BitTorrent::TorrentState state)
     case BitTorrent::TorrentState::DownloadingMetadata:
     case BitTorrent::TorrentState::ForcedDownloadingMetadata:
         if (!dark)
-            return {34, 139, 34}; // Forest Green
+            return {15, 187, 88}; // Deep Green
         else
-            return {50, 205, 50}; // Lime Green
+            return {108, 194, 74}; // Light Green
     case BitTorrent::TorrentState::StalledDownloading:
     case BitTorrent::TorrentState::StalledUploading:
         if (!dark)
             return {0, 0, 0}; // Black
         else
-            return {204, 204, 204}; // Gray 80
+            return {104, 104, 104}; // Grey
```

Comment:
Too dark for dark themes IMO

![173026](https://user-images.githubusercontent.com/6451685/156011067-8c1e9ebd-6b35-4f23-a8cc-e3eb291543c1.png)

Also not to comment in another line, completed and active seeding have the exact same color (blue)?

Reply:
Checkout changes.

Reply:
Now Completed has the same color as Downloading... Other than that, everything LGTM.